CMakeLists.txt

```diff
@@ -236,53 +181,24 @@ endif()
 
 set_project_standards(_CuraEngine)
 enable_sanitizers(_CuraEngine)
-set_rpath(TARGETS
-        _CuraEngine
-        PATHS
-        "$<$<PLATFORM_ID:Linux>:usr/bin>"
-        "$<$<PLATFORM_ID:Linux>:usr/bin/lib>"
-        "$<$<PLATFORM_ID:Darwin>:../lib>"
-        RELATIVE)
 
 if(${EXTENSIVE_WARNINGS})
     set_project_warnings(_CuraEngine)
 endif()
 
-if(NOT APPLE AND ENABLE_OPENMP)
-    find_package(OpenMP REQUIRED)
-    target_link_libraries(_CuraEngine PUBLIC OpenMP::OpenMP_CXX)
+if(ENABLE_OPENMP)
+    find_package(llvm-openmp REQUIRED)
+    target_link_libraries(_CuraEngine PUBLIC llvm-openmp::llvm-openmp)
 endif()
 
 if(ENABLE_ARCUS)
-    target_link_libraries(_CuraEngine PRIVATE Arcus protobuf::protobuf)
+    target_link_libraries(_CuraEngine PRIVATE arcus::libarcus protobuf::libprotobuf)
 endif()
 
-find_package(clipper 6.4.2 QUIET)
-if (NOT TARGET clipper::clipper)
-    add_library(clipper::clipper STATIC libs/clipper/clipper.cpp)
-    target_include_directories(clipper::clipper
-            PUBLIC
-            $<BUILD_INTERFACE:${CMAKE_SOURCE_DIR}/libs/clipper/include>
-            $<INSTALL_INTERFACE:${CMAKE_INSTALL_INCLUDEDIR}>
-            )
-endif ()
-
-find_package(rapidjson 1.1.0 QUIET)
-if (NOT TARGET rapidjson::rapidjson)
-    add_library(rapidjson::rapidjson INTERFACE)
-    target_include_directories(rapidjson::rapidjson
-            PUBLIC
-            $<BUILD_INTERFACE:${CMAKE_SOURCE_DIR}/libs/clipper/include>
-            $<INSTALL_INTERFACE:${CMAKE_INSTALL_INCLUDEDIR}>
-            )
-endif()
-
-find_package(stb REQUIRED)
-
-find_package(Boost REQUIRED 1.75.0)
-if (NOT TARGET Boost::boost)  # Workaround for different namespace in non Conan find_packages
-    add_library(boost::boost ALIAS Boost::boost)
-endif ()
+find_package(clipper 6.4.2 REQUIRED)
+find_package(rapidjson 1.1.0 REQUIRED)
+find_package(stb 20200203 REQUIRED)
+find_package(Boost 1.78.0 REQUIRED)
```

Comment:
Another sneaky version upgrade.

Reply:
We use 1.78.0 since version Cura 5.0.0 https://github.com/Ultimaker/cura-build-environment/blob/a8a4b6b169ac8024fbdaf17f8f0b1e9f88521f24/conanfile.py#L23

Check the conanfile.py in this repo for the actual used version